docs/04_web/05_micro-front-ends/03_foundation-header/03_foundation-header.md

```diff
@@ -26,18 +26,15 @@ You can customise:
 - the icon shown on the navigation bar and flyout menu (this shows the Genesis logo by default)
 - navigation links at the left-hand side of the navigation bar
 - the control buttons on the right-hand side of the navigation bar; these can be shown or hidden, and their behaviour controlled via event listeners
-- the contents of the flyout menu
+- The contents of the flyout menu
```

Comment:
question:

should we capitalize the other items?

Reply:
I have just synced the docs from the UI repo which already has reviews. I am personally going to leave this as it is but if the docs team want to change this then happy with that too (and then will resync with UI repo)

Reply:
Check the entry on Lists in the style guide
https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b#b305f37d9b1d44e1bafeeabd06894b12